src/NuGet.Core/NuGet.Protocol.Core.v3/Resources/PackageUpdateResource.cs

```diff
@@ -24,6 +25,13 @@ namespace NuGet.Protocol.Core.Types
     public class PackageUpdateResource : INuGetResource
     {
         private const string ServiceEndpoint = "/api/v2/package";
+        private const string ApiKeyHeader = "X-NuGet-ApiKey";
+        private const string InvalidApiKey = "invalidapikey";
+
+        /// <summary>
+        /// Create temporary verification api key endpoint: "create-verification-key/[package id]/[package version]"
+        /// </summary>
+        private const string TempApiKeyServiceEndpoint = "create-verification-key/{0}/{1}";
```

Comment:
Per discussion today, need to remove version from create-verification-key endpoint